tests/parametric/test_otel_span_methods.py

```diff
@@ -286,3 +286,774 @@ def test_otel_get_span_context(self, test_agent, test_library):
                     assert context.get("trace_id") == parent.span_context().get("trace_id")
                     assert context.get("span_id") == "{:016x}".format(span.span_id)
                     assert context.get("trace_flags") == "01"
+
+    @missing_feature(context.library == "go", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    def test_otel_span_operation_name_http_server(self, test_agent, test_library):
+        """
+            Tests that the operation name will be set to "http.server.request" when:
+            - Span kind is set to Server
+            - http.request.method is set to something (e.g., GET in this example)
+
+            (https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/http/)
+        """
+        with test_library:
+            with test_library.otel_start_span("otel_span_name", span_kind=SK_SERVER) as span:
+                span.set_attributes({"http.request.method": "GET"})
+                span.end_span()
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, OtelSpan(name="otel_span_name"))
```

Comment:
Have you tried the test on your tracer?
Here, you are looking for similar span by comparing OTel span names **which are resource names** to DD span names **which are operation names.**

How is this supposed to work? 🤔

Reply:
Thanks for catching this and fixing it!